### PR TITLE
#151168131 send invites other users

### PR DESCRIPTION
**Pull request by nebanat**
#### What does this PR do?
Users are able to search for other users and send them game invite via Email.
#### Description of Task to be completed?
- When a user starts a game and wants to invite other friends to join the game
- User can click on the 'invite other users' button to see a modal with a search box
- User can then search and send invites to other users
- Invited users can see link to join the game in their emails
#### How should this be manually tested?
- On the main gaming page users should be able search and send invites to friends
#### Any background context you want to provide?
Used nodemailer to send emails. Nodemailer is an effective node package for sending in-app notifications in node application 
#### What are the relevant pivotal tracker stories?
Story type: Feature
Story Id: 151168131
Story: Users should be able to send invites to at most 11 people other than themselves
#### Screenshots 
screenshot1
screenshots 2
screenshots 3

#### Questions:

### CODE REVIEW

File: app/controllers/users.js [hunk 1]
```diff
@@ -1,21 +1,90 @@
...
+// send email invite to users
+exports.sendInviteEmail = (req, res) => {
+  const { emails, message } = req.body;
+  // creates a transporter to send email //
+  const transporter = nodemailer.createTransport({
+    host: 'smtp.gmail.com',
+    port: 587,
+    secure: false,
+    auth: {
+      user: 'abiliyok@gmail.com',
```
Comment (ibonly) [inline app/controllers/users.js]: This is not a good practice, pass this through your environment variable.

File: .env [hunk 2]
```diff
@@ -1 +1 @@
-MONGOHQ_URL=mongo://localhost:27017/cfh
\ No newline at end of file
+MONGOHQ_URL=mongodb://localhost:27017/cfh
```
Comment (ibonly) [inline .env]: This file should be deleted, .env file is specific to the application configuration on your local machine!!!

**Review by ibonly — CHANGES_REQUESTED** [review]
Get 2 more reviewers when you fix the review.

**Review by enodi — APPROVED** [review]

**Review by dinobi — APPROVED** [review]
Seen

**Review by ibonly — APPROVED** [review]

**Pull request merged**